src/app/pages/MostReadPage/index.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+import { string, node } from 'prop-types';
+import {
+  GEL_GROUP_2_SCREEN_WIDTH_MIN,
+  GEL_GROUP_3_SCREEN_WIDTH_MAX,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MAX,
+  GEL_GROUP_5_SCREEN_WIDTH_MIN,
+} from '@bbc/gel-foundations/breakpoints';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+  GEL_SPACING_QUIN,
+} from '@bbc/gel-foundations/spacings';
+import { C_METAL } from '@bbc/psammead-styles/colours';
+import { getParagon } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { ServiceContext } from '#contexts/ServiceContext';
+import MostReadContainer from '#containers/MostRead';
+import mostReadShape from '#containers/MostRead/utilities/mostReadShape';
+import ATIAnalytics from '#containers/ATIAnalytics';
+import ChartbeatAnalytics from '#containers/ChartbeatAnalytics';
+import MetadataContainer from '#containers/Metadata';
+import Grid, { GelPageGrid } from '#app/components/Grid';
+
+const StyledMain = styled.main.attrs({ role: 'main' })`
+  flex-grow: 1;
+  margin: 0 ${GEL_SPACING};
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
+    margin: 0 ${GEL_SPACING_DBL};
+  }
+`;
+
+const ConstrainedWrapper = styled.div`
+  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    margin-bottom: ${GEL_SPACING_TRPL};
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+    margin-bottom: ${GEL_SPACING_QUIN};
+  }
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    width: 100%; /* Needed for IE11 */
+    margin: 0 auto ${GEL_SPACING_QUIN};
+    max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
+  }
+`;
+
+const HeadingOne = styled.h1.attrs({
+  id: 'content',
+  tabIndex: '-1',
+})`
+  color: ${C_METAL};
+  ${({ script }) => script && getParagon(script)};
+  ${({ service }) => getSansRegular(service)};
+  margin: 0;
+  padding: ${GEL_SPACING_TRPL} 0 ${GEL_SPACING};
+`;
+
+const StyledGelPageGrid = styled(GelPageGrid)`
+  flex-grow: 1;
```

Comment:
Do you need this?

Reply:
Just tested it locally without this flex-grow and it still works as expected 👍 Will remove now